bin/varnishtest/vtc.c

```diff
@@ -746,7 +760,8 @@ cmd_feature(CMD_ARGS)
 				vtc_stop = 2;
 			}
 #endif
-		} else if (!strcmp(*av, "cmd")) {
+		}
+		else if (!strcmp(*av, "cmd")) {
```

Comment:
Really? In the next hunk further down, that would look like this:
```
                        else
                                vtc_stop = 2;
                } else if (!strcmp(*av, "ignore_unknown_macro")) {
                        ign_unknown_macro = 1;
                        good = 1;
                }
```
Which looked too crowded and ugly to me -- you can hardly see the separation from the if block. So I put the else if on a separate line, and changed this line only so that the syntax would be consistent.

But all right then, if that's the house style, that's what we'll do.

(Not that it matters, since it's the house style, but I despise else on the same line as the closing brace for if.)